Stripe/PublicHeaders/STPPaymentMethodIdeal.h

```diff
@@ -0,0 +1,29 @@
+//
+//  STPPaymentMethodIdeal.h
+//  Stripe
+//
+//  Created by Yuki Tokuhiro on 3/9/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPPaymentMethodIdeal : NSObject <STPAPIResponseDecodable>
+
+/**
+ The customer’s bank.
+ */
+@property (nonatomic, nullable, readonly) NSString *bank;
```

Comment:
Is this the name of the bank? If so can we rename to reflect (or so it's more descriptive of whatever it is) to something like `bankName`?